pkg/bkctl/autorecovery/get_lost_bookie_recovery_delay.go

```diff
@@ -0,0 +1,64 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package autorecovery
+
+import (
+	"github.com/streamnative/pulsarctl/pkg/cmdutils"
+)
+
+func getLostBookieRecoveryDelayCmd(vc *cmdutils.VerbCmd) {
+	var desc cmdutils.LongDescription
+	desc.CommandUsedFor = "This command is used for getting the lost bookie recovery delay in second of a bookie."
+	desc.CommandPermission = "none"
+
+	var examples []cmdutils.Example
+	get := cmdutils.Example{
+		Desc:    "Get the lost Bookie Recovery Delay of a bookie",
+		Command: "pulsarctl bookkeeeper autorecovery getdelay",
+	}
+	examples = append(examples, get)
+	desc.CommandExamples = examples
+
+	var out []cmdutils.Output
+	successOut := cmdutils.Output{
+		Desc: "normal output",
+		Out:  "lostBookieRecoveryDelay value: (delay)",
+	}
+	out = append(out, successOut)
+	desc.CommandOutput = out
+
+	vc.SetDescription(
+		"getdelay",
```

Comment:
`getdelay` is a very ambiguous verb. Please use "get-lost-bookie-recovery-delay"